Fix russian locale.  Closes: #368655.

This aligns with the glibc definition in /usr/share/locale/locale.alias.
For the same glib reason, we cannot change ru and ru_RU away from ISO8859-5,
since that encoding is enshrined in /usr/share/i18n/locales/ru_RU.  Insert 
appropriate perjorative language here.

Suggestion: can the russian default be changed to UTF-8 instead of KOI8-R?  
This will make it easier for russians to fit in with the rest of the world.

Index: libx11/nls/locale.alias.pre
===================================================================
--- libx11.orig/nls/locale.alias.pre	2006-11-18 22:41:05.000000000 +1100
+++ libx11/nls/locale.alias.pre	2006-11-18 22:41:46.000000000 +1100
@@ -1183,7 +1183,7 @@
 portuguese.iso88591:				pt_PT.ISO8859-1
 romanian:					ro_RO.ISO8859-2
 rumanian:					ro_RO.ISO8859-2
-russian:					ru_RU.ISO8859-5
+russian:					ru_RU.KOI8-R
 serbocroatian:					sr_CS.ISO8859-2
 sinhala:					si_LK.UTF-8
 slovak:						sk_SK.ISO8859-2
